packages/application-generic/src/usecases/get-novu-integration/get-novu-integration.usecase.ts

```diff
@@ -26,14 +30,28 @@ export class GetNovuIntegration {
   async execute(
     command: GetNovuIntegrationCommand
   ): Promise<IntegrationEntity | undefined> {
-    if (!process.env.NOVU_EMAIL_INTEGRATION_API_KEY || !command.channelType) {
+    const channelType = command.channelType;
+
+    if (
+      channelType === ChannelTypeEnum.EMAIL &&
+      !process.env.NOVU_EMAIL_INTEGRATION_API_KEY
+    ) {
+      return;
+    }
+
+    if (
+      channelType === ChannelTypeEnum.SMS &&
+      !process.env.NOVU_SMS_INTEGRATION_ACCOUNT_SID &&
+      !process.env.NOVU_SMS_INTEGRATION_TOKEN &&
+      !process.env.NOVU_SMS_INTEGRATION_SENDER
+    ) {
```

Comment:
This validation checks are used in 2 different use cases. We need to find a way to isolate them in one single place and reuse them.